executor/show.go

```diff
@@ -166,11 +166,28 @@ func (e *ShowExec) fetchShowEngines() error {
 	return nil
 }
 
+// move information_schema to the first, and the other are sorted in the origin ascending order.
```

Comment:
s/ move/ moveInfoSchemaToFront moves
s/ the other/ the others

Reply:
Good!